src/pages/en/guides/server-side-rendering.mdx

```diff
@@ -117,6 +117,20 @@ export const prerender = true;
 </html>
 ```
 
+[MDX pages](/en/guides/markdown-content/#using-exported-variables-in-mdx) can also be prerendered:
+
+```astro title="src/pages/markdown.mdx" {2}
+---
+layout: '../layouts/markdown.astro'
+title: 'My page'
+---
+export const prerender = true;
+
+# This is my page
+```
+
+Plain `.md` Markdown pages do not support exporting variables, and so must be converted to `.mdx` before they can be prerendered.
```

Comment:
```suggestion
Markdown (`.md`) pages do not support prerendering.
```
We don't need to explicitly mention converting - it's clear that they must choose one of the other options in order to use prerendering.

Reply:
Yup, just want to say I agree with Dan here. I never like to tell people to change their stuff. (We have some VERY strong fans of Markdown :sweat_smile:  and you don't necessarily want to make it sound like THEY made a poor choice or something.) This gives them the information they need so they can decide whether it's worth the tradeoff for themselves!